corehq/apps/locations/management/commands/set_location_id.py

```diff
@@ -0,0 +1,34 @@
+from django.core.management.base import BaseCommand
+from dimagi.utils.couch.database import iter_docs
+from corehq.apps.users.models import CouchUser, CommCareUser
+
+
+class Command(BaseCommand):
+    help = ''
+
+    def handle(self, *args, **options):
```

Comment:
this needs to be run after deploy to set the location id instead of dropping the ID in a list